app/assets/stylesheets/wishlist.scss

```diff
@@ -0,0 +1,26 @@
+.wish-products-list {
+  height: 50em;
+}
+
+.border-wrap-grey-small {
+  border-bottom: 1px solid $grey-border;
+}
+
+.bigger-font {
+  font-size: 1.05em;
+}
```

Comment:
I think the color ```$grey-border``` could be renamed so it can be reusable for something that's not a border.